### DISCUSSION TITLE
Add tables with parent-child relation

### DISCUSSION THREAD

**Discussion by avazon** [AmazingSoftwareSolutions/Amazing.Oqtane.Studio#42] [Ideas]
Great work and thanks for sharing info!
I would find it useful to create tables with parent-child relation which I found troublesome to do manually in oqtane. An example would be invoices in parent table with invoice rows in a child table.

**Comment by vnetonline** [comment]
Thanks, Yes I am planning to include that functionality in [1.0.5](https://github.com/AmazingSoftwareSolutions/Amazing.Oqtane.Studio/milestone/6) schedule to be released at the end of this week
